go.mod

```diff
@@ -107,3 +107,5 @@ require (
 replace mellium.im/sasl => github.com/mellium/sasl v0.2.1
 
 replace github.com/onflow/flow-go/crypto => ./crypto
+
+replace github.com/onflow/cadence => github.com/dapperlabs/cadence-internal v0.21.0-patch.1
```

Comment:
is this something that will be switched back to the non-internal public facing repo before the mainnet spork on Feb 9th? Some operators compile code from scratch and have complained about reference to internal repo in the past :(

Reply:
Yes, once the security patch is deployed to the networks, we will port them to the public repository. We are aware of the problem of being unable to compiling the code from source, but we cannot make security fixes public before they are not deployed.

We try to get the fixes in as part of the regularly scheduled sporks to avoid additional ad-hoc / unplanned software updates that need to be performed in coordination. If it would be easier to do deploy the fix outside of the spork schedule, then we should consider changing the current approach.